src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -4113,9 +4113,25 @@ private function step_in_foreign_content(): bool {
 				return true;
 
 			/*
-			 * > A comment token
+			 * A CDATA section is effectively a text node and should have similar handling.
 			 */
 			case '#cdata-section':
+				/*
+				 * NULL bytes and whitespace do not change the frameset-ok flag.
+				 */
+				$current_token        = $this->bookmarks[ $this->state->current_token->bookmark_name ];
+				$cdata_content_start  = $current_token->start + 9;
+				$cdata_content_length = $current_token->length - 12;
+				if ( $cdata_content_length !== strspn( $this->html, "\0 \t\n\f\r", $cdata_content_start, $cdata_content_length ) ) {
```

Comment:
The yoda condition PHPCS flagged here seems to be a false positive. I suspect it's confusing a string literal on the line with a literal comparison, but the string literal _is not_ being compared. It's fine to have it on the other side…

Reply:
I _suspect_ it was finding a literal in the RHS of the comparison and thinking that literal are supposed to go on the left (yoda), but it ignores the fact that the literal is in a function call. But that's just my intuition.

Switching the order is fine if it satisfies the linter 🤷

Reply:
I believe it's the variable on the left hand side that it doesn't like, unless both are variables, in which case it's happy and doesn't care.

joda condition checks like these aren't about literals, but accidental assignment to a variable.